src/main/java/com/webapp/TextBook/repository/shared_respository_helper/DataAccessConversionHelper.java

```diff
@@ -0,0 +1,44 @@
+package com.webapp.TextBook.repository.shared_respository_helper;
+
+import com.webapp.TextBook.repository.DataAccessConversion;
+import com.webapp.TextBook.repository.DataAccessConversionException;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+import java.util.function.Supplier;
+
```

Comment:
documentation and commentation is needed here

Reply:
a9a58c57a98eb3e18a62caaa9a643812ad605318